src/Templating/Helper/BlockHelper.php

```diff
@@ -160,8 +160,15 @@ public function renderEvent($name, array $options = [])
     {
         $eventName = sprintf('sonata.block.event.%s', $name);
 
-        /** @var BlockEvent $event */
-        $event = $this->eventDispatcher->dispatch($eventName, new BlockEvent($options));
+        $reflectionMethod = new \ReflectionMethod($this->eventDispatcher, 'dispatch');
```

Comment:
Can you please add a `NEXT_MAJOR` note

Reply:
What do I need to add ?
Next major:  Remove this ?
Does this mean you'll drop the support of symfony 2 and 3 ?

Reply:
Something like:
```
NEXT_MAJOR: remove this when dropping support for symfony 3.x
```